### PR TITLE
Lightwave

### PR DESCRIPTION
**Pull request by ColinRobbins**
## Proposed change

Added support for Lightwave TRVs
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: [Pull 31665](https://github.com/home-assistant/home-assistant/pull/31665)
- This PR fixes or closes issue:

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
Configuration block for this integration is missing:

<https://developers.home-assistant.io/docs/en/documentation_create_page.html#configuration>

Please add it 👍

**Comment by ColinRobbins** [comment]
Configuration block added (NB: it was missing before my edits!)

**Comment by frenck** [comment]
I know it was missing, nevertheless, considering new options, I think it is helpful to add it right now. Just to prevent future confusion with our users.

**Comment by frenck** [comment]
Can you please make sure the build passes? Thanks 👍

**Comment by ColinRobbins** [comment]
Opps - sorry.  Done.

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

File: source/_integrations/lightwave.markdown [hunk 1]
```diff
@@ -37,14 +37,84 @@ lightwave:
...
+  description: List of switches you wish to configure
+  required: false
+  type: map
+  keys:
+    name:
+      description: Name of the Switch
+      required: true
+      type: string
+trv:
+  description: PLEASE ADD A DESCRIPTION HERE :)
```
Comment (klaasnicolaas) [inline source/_integrations/lightwave.markdown]: @ColinRobbins Could you update the description here?

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]
I have adjusted it a bit because otherwise it will not be due to the jykell build, see my review below and what you still need to adjust.

File: source/_integrations/lightwave.markdown (hunk 1 above)
Comment (ColinRobbins) [reply source/_integrations/lightwave.markdown]: I've made this change...

**Review by frenck — APPROVED** [review]

**Pull request merged**